tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["packages", "scripts", "./*.ts"]
```

Comment:
The `tsconfig.eslint.json` added here is to prevent ESLint from reporting errors for files that are not matched. Please refer to: https://typescript-eslint.io/linting/troubleshooting/#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file